src/index.html

```diff
@@ -3,11 +3,60 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
```

Comment:
такое же исправление было в предыдущем задании, внимательнее
```suggestion
    <meta 
      name="viewport"
```